cmd/clusterctl/examples/openstack/provider-component/cluster-api/kustomization.yaml

```diff
@@ -4,14 +4,14 @@ generatorOptions:
   disableNameSuffixHash: true
 
```

Comment:
I replaced the resources block through the following. This includes the kustomization files in this folder which should lead to the same result (but we don't need the cp in generate-yaml.sh):
````
bases:
-  ../../../../../../vendor/sigs.k8s.io/cluster-api/config/rbac
- ../../../../../../vendor/sigs.k8s.io/cluster-api/config/crds
- ../../../../../../vendor/sigs.k8s.io/cluster-api/config/manager
````

Reply:
the issue is I tried this and I was informed by kustomize that include files outside the folder violate the security restriction, so I have to copy the files...

Reply:
I have to check. That was in the release notes
> In 2.0, bases can continue to specify, via relative paths, kustomizations outside the current kustomization directory. But non-base paths are constrained to terminate in or below the current kustomization directory. 

https://github.com/kubernetes-sigs/kustomize/blob/master/docs/v_2.0.0.md

And it looked like it worked